main/mimi.c

```
@@ -1,190 +1,166 @@
-            ESP_LOGW(TAG, "WiFi connection timeout");
-        }
-    } else {
-        ESP_LOGW(TAG, "No WiFi credentials configured");
+
+    /* LLM 初始化只做静态检查，不会真正发请求。 */
+    err = llm_proxy_init();
+    if (err != ESP_OK) {
+        ESP_LOGE(TAG, "LLM init failed: %s", esp_err_to_name(err));
+        print_block("MVP ERROR", "LLM 初始化失败，请检查 mvp_config.h 里的 DeepSeek API Key 和 endpoint。");
+        delay_and_idle();
+        return;
```

Cppcheck (2.20.0)

[error] Memory leak

(memleak)